go/apps/api/openapi/spec/paths/_internal/vault/decrypt/index.yaml

```
@@ -0,0 +1,37 @@
+    "400":
+      content:
+        application/json:
+          schema:
+            $ref: "../../../../error/BadRequestErrorResponse.yaml"
+      description: Invalid request body
+    "500":
+      content:
+        application/json:
+          schema:
+            $ref: "../../../../error/InternalServerErrorResponse.yaml"
+      description: Internal server error during decryption
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)